esi_leap/tests/api/controllers/v1/test_offer.py

```diff
@@ -39,3 +40,9 @@ def test_one(self):
         o = create_test_offer(self.context)
         data = self.get_json('/offers')
         self.assertEqual(o.uuid, data['offers'][0]["uuid"])
+
+    def test_get_invalid_time(self):
+
+        start = datetime.datetime(2016, 7, 16, 19, 20, 30)
+
+        self.get_json('/offers?start_time=' + str(start), expect_errors=True)
```

Comment:
We should probably test this slightly more, as in https://opendev.org/openstack/ironic/src/branch/master/ironic/tests/unit/api/controllers/v1/test_node.py#L455-L465